packages/models/src/errors.ts

```diff
@@ -431,3 +432,13 @@ export function invalidKey(
     title: "Invalid Key",
   });
 }
+
+export function featureFlagNotEnabled(
+  featureFlag: string
+): ApiError<CommonErrorCodes> {
+  return new ApiError({
+    detail: `Feature flag ${featureFlag} is not enabled`,
+    code: "featureFlagNotEnabled",
+    title: "Feature flag not enabled",
+  });
```

Comment:
This error will potentially arrive to the end user trought APIV2, maybe we could think of a more user friendly description? Idk, @ecamellini what do you think?

Reply:
I think at the moment it's ok to leave it like this, in general if a feature is not enabled it will probably not be exposed from BFF/API GW, we can think about it better when reviewing errors in API v2 developments